modules/BrowserHistory.js

```diff
@@ -106,6 +108,10 @@ class BrowserHistory extends DOMHistory {
     }
   }
 
+  static get history() {
```

Comment:
People who need to support IE8 (poor folks) are going to gripe about this. Let's just `export history new BrowserHistory` in this module instead. Then people can still `import { history }` and we'll be fine.